genie-web/src/main/java/com/netflix/genie/web/jobs/workflow/impl/JobKickoffTask.java

```diff
@@ -152,11 +151,6 @@ public void executeTask(@NotNull final Map<String, Object> context) throws Genie
                 .redirectOutput(new File(jobExecEnv.getJobWorkingDir() + JobConstants.GENIE_LOG_PATH))
                 .redirectError(new File(jobExecEnv.getJobWorkingDir() + JobConstants.GENIE_LOG_PATH));
 
-            //
-            // Check if file can be executed. This is to fix issue where execution of the run script fails because
```

Comment:
We're sure this isn't an issue still?

Reply:
I really don't see how `touch` would have made any difference.

But i did deploy to my cluster and run the smoke tests.